src/Serilog.Settings.Configuration/Settings/Configuration/ConfigurationReader.cs

```diff
@@ -50,6 +51,63 @@ public void Configure(LoggerConfiguration loggerConfiguration)
             ApplyAuditSinks(loggerConfiguration);
         }
 
+        void ProcessFilterSwitchDeclarations()
+        {
+            var filterSwitchesDirective = _section.GetSection("FilterSwitches");
+
+            foreach (var filterSwitchDeclaration in filterSwitchesDirective.GetChildren())
+            {
+                var filterSwitch = LoggingFilterSwitchProxy.Create();
+                if (filterSwitch == null)
+                {
+                    SelfLog.WriteLine($"FilterSwitches section found, but Serilog.Filters.Expressions isn't referenced.");
```

Comment:
_... neither Serilog.Expressions nor Serilog.Filters.Expressions is referenced._ ?